runtime/ddr/dummy_headers/unistd.h

```diff
@@ -19,3 +19,11 @@
  *
  * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
  *******************************************************************************/
+
+#ifndef UNISTD_H
+#define UNISTD_H
+
+/* Assume we have posix priority scheduling, so omr
+   doesn't complain it can't implement THREAD_YIELD_NEW */
```

Comment:
Please fix this comment:
* POSIX and OMR should be upper-case
* the sentence should end with a period
* multi-line comments should begin each line with `*`
* the end of the comment should be on a line by itself
```
/*
 * Assume we have POSIX priority scheduling, so OMR
 * doesn't complain it can't implement THREAD_YIELD_NEW.
 */
```